tests/unit/test_job.py

```diff
@@ -255,6 +305,50 @@ def test_job_request_with_qprogram(user: User, simulator_device: Device):
 
     assert isinstance(job, Job)
     assert job.job_request == expected_job_request
+    assert all(
+        [
+            "data" in job.job_request.description,
```

Comment:
This is only assert if [True True True], right?

Reply:
yep, `all([item, item, item])` should be equivalent to `item and item and item`. I tend to use the `all` pattern for long lists.